tests/manifests/modified_common.yaml

```diff
@@ -23,57 +23,12 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims"]
     verbs: ["get", "list", "watch", "update", "delete", "create","patch"]
-  - apiGroups: ["storage.k8s.io"]
```

Comment:
Looks good to remove all the unnecessary permissions. Separately from this PR, we should really rename it from the toolbox to the migrator. Then it's clear this rbac and the migrator pod just needs to be created during the migration, then it can be deleted when they are done.

Reply:
created follow up PR #22